src/test/scala/Main.scala

```diff
@@ -0,0 +1,49 @@
+package scalaz
+
+package schema
+
+import scala.concurrent.{ Await, Future }
+import scala.concurrent.duration.Duration
+import scala.concurrent.ExecutionContext.global
+
+import testz._
+import testz.runner._
+
+import scala.Console._
+
+object Main {
+
+  def printReport(scope: List[String], out: Result): List[String] =
+    "\n" :: (out match {
+      case Succeed => Runner.fastConcatDelim(new ::(s"[${GREEN}OK${RESET}]\n", scope), " > ")
+      case fail: Fail =>
+        Runner.fastConcatDelim(new ::(s"[${RED}KO${RESET}]\n", scope), " > ") ++ fail.failures.map {
+          case Left(t)    => t.getMessage
+          case Right(err) => err
+        }
+    })
+
+  val harness: Harness[PureHarness.Uses[Unit]] =
+    PureHarness.toHarness(
+      PureHarness.make(
+        (ls, tr) => Runner.printStrs(printReport(ls, tr), print)
+      )
+    )
+
+  def tests[T](harness: Harness[T]): List[(String, T)] =
+    List(
+      ("Examples", SchemaModuleExamples.tests(harness))
+    )
+
+  def suites(harness: Harness[PureHarness.Uses[Unit]]): List[() => Future[TestOutput]] =
+    tests(harness).map {
+      case (name, suite) =>
+        () => Future.successful(suite((), List(name)))
+    }
+
+  def main(args: Array[String]): Unit = {
+    val result = Await.result(Runner(suites(harness), global), Duration.Inf)
```

Comment:
In case of non-terminating test, will Travis kill job after some timeout or we'll have to call someone with permissions?

Reply:
Good question. I suppose Travis has some kind of global timeout, but I'll double check that

Reply:
So, as mentioned [here](https://docs.travis-ci.com/user/common-build-problems/#my-builds-are-timing-out) travis builds will timeout after 10 minutes without producing any output.